rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1029,6 +1032,11 @@ stream {
             set $service_port   "{{ $location.Port }}";
             set $location_path  "{{ $location.Path | escapeLiteralDollar }}";
 
+            {{/* check the location is not using the upstream-vhost annotation */}}
```

Comment:
What do you mean? The if condition below is actually checking if the location _is_ using the upstream-vhost annotation.

IMHO even the fixed version of this comment does not add any value, the logic is quite readable and not complex.

Reply:
comment removed